### DISCUSSION TITLE
impossible to "ionic capacitor build ios"

### DISCUSSION THREAD

**Discussion by aragon5956** [ionic-team/capacitor#6578] [General]
impossible to do this command i have this error :
`npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: FacilNdf@0.0.1
npm ERR! Found: @capacitor/core@5.0.1
npm ERR! node_modules/@capacitor/core
npm ERR!   @capacitor/core@"^5.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @capacitor/core@"^4.7.0" from @capacitor/ios@4.8.0
npm ERR! node_modules/@capacitor/ios
npm ERR!   @capacitor/ios@"4.8.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/alexandre/.npm/_logs/2023-05-08T14_31_17_285Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/alexandre/.npm/_logs/2023-05-08T14_31_17_285Z-debug-0.log
[ERROR] An error occurred while running subprocess npm.
        
        npm i -E @capacitor/ios@4.8.0 exited with exit code 1.
        
        Re-running this command with the --verbose flag may provide more information.
`
could you help me ?

**Comment by jcesarmobile** [comment]
Looks like you are mixing Capacitor 4.x packages with Capacitor 5.x packages. You should not mix them, if you use Capacitor 4, then all your @capacitor packages should be 4.x, if you use Capacitor 5, then all your @capacitor packages should be 5.x.

**Comment by aragon5956** [comment]
why the package is not 5.x ?

**Reply by jcesarmobile** [reply]
It's your project, so you should know better, I have not enough information to know what's happening in your project, you don't even mention which command you are running

**Reply by kensodemann** [reply]
I suggest reviewing your Capacitor plugins and ensuring they have all been updated to support Capacitor v5. I am going to guess one of the community-based plugins being used in your project has not been updated.

**Comment by aragon5956** [comment]
hahahahhahah

**Comment by aragon5956** [comment]
could you upgrade the version of the packages to 5.x please ?

**Reply by jcesarmobile** [reply]
from your app? sure, give ssh access to your computer

**Reply by aragon5956** [reply]
the 5.x version of capacitor/ios doesn't exist , that's i want , please ,i 've problem for this command

**Comment by aragon5956** [comment]
ok it's done !! thanks you !
